### DISCUSSION TITLE
Wiki.js/dropbox: Where are files stored?

### DISCUSSION THREAD

**Discussion by wallyreport** [requarks/wiki#6476] [Help / Questions]
If I install wiki.js with a dropbox backend, are files stored locally, in text format, in my dropbox folder?

**Comment by NGPixel** [comment]
There's no dropbox backend...

**Comment by wallyreport** [comment]
I thought the "Dropbox Strategy" described in the link below was about using Wiki.js to access dropbox files. Which, I would think, is essentially using dropbox as a backend. 
[https://docs.requarks.io/auth/dropbox](https://docs.requarks.io/auth/dropbox)

**Reply by NGPixel** [reply]
That's for authentication, not storing data.
